sklearn/multiclass.py

```diff
@@ -74,6 +76,15 @@ def _fit_binary(estimator, X, y, classes=None):
     return estimator
 
 
+def _partial_fit_binary(estimator, X, y, classes=None):
+    """Partially fit a single binary estimator."""
+    if (not hasattr(estimator, "partial_fit")):
```

Comment:
This should just be checked once and for all under the `if _check_partial_fit_first_call` block.